distributed/cli/tests/test_cli_utils.py

```diff
@@ -17,6 +17,7 @@ def test_uri_from_host_port():
     assert f('', 456, 123) == 'tcp://:456'
     assert f('', None, 123) == 'tcp://:123'
     assert f('', None, 0) == 'tcp://'
+    assert f('', 0, 123) == 'tcp://:0'
```

Comment:
Yes, what I mean is that `uri_from_host_port` can return `tcp://` here, it doesn't have to return `tcp://:0`.